test/executor/copy_test.cpp

```diff
@@ -120,8 +120,7 @@ TEST_F(CopyTests, Copying) {
   auto copy_stmt = peloton_parser.BuildParseTree(copy_sql);
 
   LOG_TRACE("Building plan tree...");
-  auto copy_plan =
-      optimizer->BuildPelotonPlanTree(copy_stmt, DEFAULT_DB_NAME, txn);
+  auto copy_plan = optimizer->BuildPelotonPlanTree(copy_stmt, txn);
```

Comment:
Do we want to bind the parse tree first in this test?

Reply:
Yes done thanks. I am actually curious why this test passes without binding.